cli/kubectl_kadalu/storage_add_parser.py

```diff
@@ -0,0 +1,363 @@
+"""
+Storage Add alternate syntax parser
+"""
+STRING = 0
```

Comment:
Initially I used this to assign unknowns, since I was checking for `:` to mark it as Storage unit or brick. But since Kadalu supports `pvc` as Storage unit, I removed that check. I will remove `STRING` as well.

Reply:
Done